src/Main.java

```diff
@@ -1,5 +1,235 @@
+            if (wordArray[i] != wordArray[j]) {
+                isPalindrom = false;
+                break;
+            }
+        }
+        if (isPalindrom == true) {
+            System.out.println("The word is palindrome!");
+        } else
+            System.out.println("The word is not a palindrome!");
+
+        // Task 5 Napravite program  kojem cemo reci opseg za koji ce morat ispisati sve parne brojeve
+        // u koje ne ulaze brojevi kojima postavljamo opseg
+        //npr
+        //Please enter lower number in range:
+        //10
+        //Please enter larger number in range:
+        //16
+        //System-Out: Even numbers for range: 10 to 16 are:
+        //   12,14
+        System.out.println("Please enter lower number in range: ");
+        int lowerRange = reader.nextInt();
+        System.out.println("Please enter larger number in range: ");
+        int largerRange = reader.nextInt();
+        String evenNumbers = "";
+        for (int i = lowerRange + 1; i < largerRange; i++) {
+            if (i % 2 == 0) {
+                evenNumbers += i + ", ";
+            }
+        }
+        System.out.println("Even numbers for range: " + lowerRange + " to " + largerRange + " are: ");
+        System.out.println(evenNumbers);
+
+        // Task 6. Napravite program koji ce objediniti tri  niza u jedan veliki niz.
+        // examples of 3 Arrey
+        // Should be populated from console
+        System.out.println("Task 6");
+        System.out.println("\n");
+        int A[] = {1, 2};
+        int B[] = {3, 4, 5, 6};
+        int C[] = {7, 8, 9};
+
+        int lengthOfD = A.length + B.length + C.length;
+        int D[] = new int[lengthOfD]; // D = { 0, 0, 0, 0, 0, 0, 0, 0, 0 }
+        // enter and populate elements
+        for (int i = 0; i < A.length; i++) {
+            D[i] = A[i];
+        } // D = { 1, 2, 0, 0, 0, 0, 0, 0, 0 }
+
+        for (int i = 0; i < B.length; i++) {
+            D[i + A.length] = B[i];
+        } // D = { 1, 2, 3, 4, 5, 6, 0, 0, 0 }
+
+        for (int i = 0; i < C.length; i++) {
+            D[i + A.length + B.length] = C[i];
+        } // D = { 1, 2, 3, 4, 5, 6, 7, 8, 9 }
+
+        for (int i = 0; i < D.length; i++) {
+            System.out.println(D[i]);
+            // D = { 1, 2, 3, 4, 5, 6, 7, 8, 9 }
+        }
+
+        // Task 7. NApravite program koji ce sabrati dva niza zajedno, ukoliko je jedan niz veci od drugo
+        // taj broj se samo prepise
+        // Npr: {1,3,5,6} , {1,1,1,1,1,1,1}
+        // Output: 2,4,6,7,1,1,1
+        System.out.println("Task 7");
+        System.out.println("\n");
+        System.out.println("Zbir nizova: {1,3,5,6} i {1,1,1,1,1,1,1} je niz: ");
+        int arrayA[] = {1,3,5,6}; // length 4
+        int arrayB[] = {1,1,1,1,1,1,1}; // length 7
+        int maxValue = max(arrayA.length, arrayB.length); // length 7
+        int[] sumArray = new int[maxValue]; // {0,0,0,0,0,0,0}
+
+        if (arrayA.length > arrayB.length)
+        {
```

Comment:
You are duplicating code, this can be done without duplicating the logic,  there is a better approach to just iterate through smaller array and add values to larger array , then you dont need to copy the numbers that are unchangegd